docs/source/examples/md/gru4rec_examples.md

```diff
@@ -1,59 +1,105 @@
-# Gru4Rec for session-based recommendation
+# Example to train gru4rec model with FedAvg Strategy
 
+This doc introduce how to use PaddleFL to train model with Fl Strategy.
 
-[Graph Convolutional Network \(GCN\)](https://arxiv.org/abs/1609.02907) is a powerful neural network designed for machine learning on graphs. Based on PGL, we reproduce GCN algorithms and reach the same level of indicators as the paper in citation network benchmarks.
-
-### Simple example to build GCN
+### Dependencies
+- paddlepaddle>=1.6
 
-To build a gcn layer, one can use our pre-defined ```pgl.layers.gcn``` or just write a gcn layer with message passing interface.
-```python
-import paddle.fluid as fluid
-def gcn_layer(graph_wrapper, node_feature, hidden_size, act):
-    def send_func(src_feat, dst_feat, edge_feat):
-        return src_feat["h"]
-    
-    def recv_func(msg):
-        return fluid.layers.sequence_pool(msg, "sum")
-    
-    message = graph_wrapper.send(send_func, nfeat_list=[("h", node_feature)])
-    output = graph_wrapper.recv(recv_func, message)
-    output = fluid.layers.fc(output, size=hidden_size, act=act)
-    return output
+### How to install PaddleFL
+please use the python which has installed paddlepaddle.
```

Comment:
please use the python which has installed paddlepaddle
please use python which has paddlepaddle installed